### DISCUSSION TITLE
Table 4 Practical 4

### DISCUSSION THREAD

**Discussion by Sambainbridge** [IBAHCM/RPiR#456] [Old]
SamBainbridge- https://github.com/IBAHCM/githubSamBainbridgeBCI.git

**Comment by ciarakg** [comment]
Ciara Goddard: ciarakg: https://github.com/IBAHCM/ciarakgBCI

**Comment by emmamclea** [comment]
EmmamcLea : https://github.com/IBAHCM/emmamcleaBCI.git
